cmd/bootstrap/cmd/partner_infos.go

```diff
@@ -0,0 +1,265 @@
+	info, err := client.ExecuteScriptAtLatestBlock(ctx, getNodeInfoScript, []cadence.Value{nodeID})
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute the get node info script: %w", err)
+	}
+
+	return info, nil
+}
+
+// parseNodeInfo convert node info retrieved from cadence script
+func parseNodeInfo(info cadence.Value) (*bootstrap.NodeInfoPub, error) {
+	fields := info.(cadence.Struct).Fields
+	nodeID, err := flow.HexStringToIdentifier(string(fields[idField].(cadence.String)))
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert flow node ID from hex string to identifier (%s): %w", string(fields[idField].(cadence.String)), err)
+	}
+
+	b, err := hex.DecodeString(string(fields[networkingKeyField].(cadence.String)))
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode network public key hex (%s): %w", string(fields[networkingKeyField].(cadence.String)), err)
+	}
+	networkPubKey, err := crypto.DecodePublicKey(crypto.ECDSAP256, b)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode network public key: %w", err)
+	}
+
+	b, err = hex.DecodeString(string(fields[stakingKeyField].(cadence.String)))
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode staking public key hex (%s): %w", string(fields[stakingKeyField].(cadence.String)), err)
+	}
+	stakingPubKey, err := crypto.DecodePublicKey(crypto.BLSBLS12381, b)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode staking public key: %w", err)
+	}
+
+	return &bootstrap.NodeInfoPub{
+		Role:          flow.Role(fields[roleField].(cadence.UInt8)),
+		Address:       string(fields[networkingAddressField].(cadence.String)),
+		NodeID:        nodeID,
+		Stake:         uint64(fields[tokensStakedField].(cadence.UFix64)),
+		NetworkPubKey: encodable.NetworkPubKey{PublicKey: networkPubKey},
+		StakingPubKey: encodable.StakingPubKey{PublicKey: stakingPubKey},
+	}, nil
+}
+
+// isFlowNode returns true if the address contains nodes.onflow.org
+func isFlowNode(address string) bool {
+	return strings.Contains(address, flowNodeAddrPart)
+}
+
+// validateANNetworkKey attempts to parse the network key provided for secure client connections
+func validateANNetworkKey(key string) error {
+	b, err := hex.DecodeString(strings.TrimPrefix(key, "0x"))
+	if err != nil {
+		return fmt.Errorf("failed to decode public key hex string: %w", err)
+	}
+
+	_, err = crypto.DecodePublicKey(crypto.ECDSAP256, b)
+	if err != nil {
+		return fmt.Errorf("failed to decode public key: %w", err)
+	}
+
+	return nil
+}
+
+// writeNodePubInfoFile writes the node-pub-info file
+func writeNodePubInfoFile(info *bootstrap.NodeInfoPub) {
+	fileOutputName := fmt.Sprintf(bootstrap.PathNodeInfoPub, info.NodeID)
+	path := filepath.Join(flagOutputDir, fileOutputName)
+	writeJSON(path, info)
+}
+
+// writePartnerStakesFile writes the partner stakes file
+func writePartnerStakesFile(partnerStakes PartnerStakesInfo) {
+	path := filepath.Join(flagOutputDir, bootstrap.PartnerStakesFileName)
```

Comment:
It seems like this variable is not attached to a flag any more 🤔

Reply:
https://github.com/onflow/flow-go/pull/1988/commits/cba109821575449a48fc29c352bf60649e5a3249